resources/react/README.md

```diff
@@ -0,0 +1,51 @@
+
+### Why React?
+<div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe width="443" height="240" src="https://www.youtube.com/embed/SH2tck0pHWI" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" style="position:absolute;top:0;left:0;width:100%;height:100%;" allowfullscreen></iframe>
+</div>
+
+## Getting Started
+
+<LevelWithButton image="/images/reactCourse.png" link="https://www.youtube.com/playlist?list=PL55RiY5tL51oyA8euSROLjMFZbXaV7skS" button="Start Learning!" desc="In this series of videos you will be introduced fundamental concepts of React as you build a small project."></LevelWithButton>
+
+<LevelWithButton :image-right=false image="https://scrimba.com/static/art/castcover.png" link="https://scrimba.com/course/glearnreact" button="Go to Tutorial!" desc="This in-depth course allows you to learn React in an interactive way at Scrimba"></LevelWithButton>
+
+<LevelWithButton image="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg" link="https://reactjs.org/" button="Read Docs" desc="These official Reactjs documentations can help you build projects if you get stuck"></LevelWithButton>
+
+## Cheat Sheet
+
+<iframe src="https://ihatetomatoes.net/wp-content/uploads/2017/01/react-cheat-sheet.pdf" width="700" height="550"></iframe>
+
+<LevelWithButton image="https://upload.wikimedia.org/wikipedia/commons/a/a7/React-icon.svg" link="https://devhints.io/react" button="Read Docs" desc="Essential Syntaxs for React from devhints.io"></LevelWithButton>
+
+## Further Steps
+We at Hacklab believe in learning by doing. So here are some ideas for a few projects that you can work on to hone your skills.
+
+### Create a simple application
+
+#### Todo Apps
+<Level desc="This is a simple Todo App that lets a user create, read, update and delete to-do tasks. In the application user can also categories the tasks as shown in the image" image="https://miro.medium.com/max/2848/1*bubW56HD3dK44Uj2Jvs-UA.png"></Level>
+
+#### SnapShot
+<Level desc="This application searches and displays the pictures that the user searches for." image="https://github.com/Yog9/SnapShot/raw/master/snapscout.png"></Level>
+
+#### Giphy
+<Level desc="Create a single page web application which uses a search input and Giphy’s API to display giphs on a page." image="https://miro.medium.com/max/1000/1*B_Ny6pA3KiFfl_9EhZfziQ.png"></Level>
+
+### Learn React Native
+<Level :image-right=false desc="React Native allows you to create native apps for Android and iOS. Entend your knowledge to make mobile applications." image="https://www.mindinventory.com/blog/wp-content/uploads/2018/11/ios-android-development-using-react-native.png"></Level>
+
+### Create your own project
+Have a good application idea, although don't know where to start from. Contact us at [Microsoft Teams](/microsoft-teams/) or meet us at room 2014 in UTM's Deerfield Hall building.
```

Comment:
If you have a good application idea, and don't know where to start, Contact us via Microsoft Teams or...